contracts/container/contract.go

```diff
@@ -1721,6 +1722,66 @@ func OwnerOf(tokenID []byte) interop.Hash160 {
 	return owner[1 : 1+interop.Hash160Len]
 }
 
+// Tokens returns iterator over IDs of all existing containers.
+//
+// Tokens implements optional NEP-11 method.
+func Tokens() iterator.Iterator {
+	return storage.Find(storage.GetReadOnlyContext(), []byte{containerKeyPrefix}, storage.KeysOnly|storage.RemovePrefix)
+}
+
+// Properties returns properties of referenced container.
+//
+// The properties are:
+//
+//	name: Base58-encoded tokenID;
```

Comment:
It can be specified in attributes as well.

Reply:
so?

Reply:
I'd use `Name` attribute in this case. Per NEP-11 `name` must be provided and that's exactly what `Name` attribute is. But the attribute is optional, so a fallback to token id is fine.